core/runtime/src/main/java/io/quarkus/runtime/logging/FileConfig.java

```diff
@@ -40,15 +45,15 @@
     String format;
 
     /**
-     * The file log level
+     * The level of logs to be written into the file.
      */
     @ConfigItem(defaultValue = "ALL")
     Level level;
 
     /**
-     * The file logging log level
+     * The name of file where logs will be stored.
```

Comment:
I would go with `The name of the file in which logs will be written.`

Reply:
Done.

Reply:
agreed, @sarxos can you change this one?

Reply:
Hi @cescoffier This this already modified in accordance to @gsmet request. Please see https://github.com/quarkusio/quarkus/pull/2589/commits/aab71c4a73481ac56de7955b65cf863c65e040b7. These two commits will be squashed after I have final approval for these changes.